cmd/aries-js-worker/scripts/build_assets.sh

```diff
@@ -6,4 +6,10 @@
 #
 
 GOOS=js GOARCH=wasm go build -o src/aries-js-worker.wasm main.go
+gzip -f src/aries-js-worker.wasm
```

Comment:
What does the CompressionPlugin now do?

Reply:
Good question - removed

Reply:
(and I guess it wasn't possible to have a common step that uses CompressionPlugin).

Reply:
Couldn't make it work without defining an entry for webpack. Couldn't quickly figure out a sensible way for it.